### ISSUE TITLE
Restoring Backup on simulator

### ISSUE THREAD

**Issue by sirjan-cn** [libimobiledevice/libimobiledevice#1031]
Is there any way using which I can restore my backup data on my Simulator?

**Comment by nikias** [comment]
No. The simulator is not exposed via usbmuxd and thus is not accessible via libimobiledevice or iTunes.
